templates/info/privacy.html.twig

```diff
@@ -6,7 +6,7 @@
 <h4 class="mb-3">Privacyverklaring {{ APP_NAME }}</h4>
 
 <p>In deze privacyverklaring lees je hoe SURFnet omgaat met je persoonsgegevens
-in het kader van het gebruik van de website <code>https://{{ APP_URLDOMAIN }}/</code> en de
+in het kader van het gebruik van de website <code>{{ APP_PROTOCOL }}://{{ APP_URLDOMAIN }}/</code> en de
```

Comment:
Make the result of this kind of glueing available as a twig globals (shorturlshort and shorturlfull) and use that here and in the other places below?